protocol/0056-REWA-rewards_overview.md

```diff
@@ -27,6 +27,8 @@ At the end of the epoch:
 1. Then the entire balance of each reward account is distributed amongst entities with a non-zero reward metric for that reward type and market using the mechanism specified in the recurring transfer.
 1. Distributed rewards are transferred to a [vesting account](./0085-RVST-rewards_vesting.md).
 
+Note, reward metrics should be calculated and exposed through an API even if the the reward pool is not to be funded this epoch (i.e. when the `transfer_interval` set in the [recurring transfer](./0057-TRAN-transfers.md#recurring-transfers-to-reward-accounts) is strictly greater than `1`).
```

Comment:
what exactly does this mean? what are you referring to by _reward metrics_ that you're expecting to be calculated and published every epoch?

Reply:
I will remove this entirely. 

It's a bit of legacy text which is somewhat redundant now we are considering publishing interim reward information.

It was supposed to ensure we were at least publishing scores each epoch even if the rewards were only distributed every n epochs.